packages/issuer/lib/api.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  CredentialFormatEnum,
+  CredentialIssuerMetadataSupportedCredentials,
+  CredentialRequest,
+  Display,
+  IssuerCredentialSubjectDisplay,
+} from '@sphereon/openid4vci-common'
+import bodyParser from 'body-parser'
+import cookieParser from 'cookie-parser'
+import * as dotenv from 'dotenv-flow'
+import express, { Express, Response } from 'express'
+import { v4 as uuidv4 } from 'uuid'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+
+import { VcIssuer } from './VcIssuer'
+import { CredentialSupportedV1_11Builder, VcIssuerBuilder } from './builder'
+import { createCredentialOfferDeeplink } from './functions'
+
+function buildVcIssuer() {
```

Comment:
buildVCIFromEnvironment, as we certainly want to have other means to instantiate issuers in the future as well

Reply:
changed this method to `buildVCIFromEnvironment` and also added another option to build `api` object with an existing metadata object